ledger/ledger.go

```diff
@@ -1128,3 +1128,24 @@ func (l *Ledger) Truncate(utxovmLastID []byte) error {
 	l.xlog.Info("truncate blockid succeed")
 	return nil
 }
+
+// StartPowBlockState 更新powBlockState状态
```

Comment:
Comments for public method should be written in English.

Reply:
I got it.